### PR TITLE
fix: Update reate-custom-builder.md and move it to the deprecated section

### PR DESCRIPTION
**Pull request by hferentschik**

### CODE REVIEW

**Comment by hferentschik** [comment]
@yelhouti, what do you think?

**Comment by ethangj** [comment]
I'm good with this, well let @jha-cloudbees approve first though

**Comment by yelhouti** [comment]
@hferentschik great, thanks :)

**Comment by hferentschik** [comment]
@jha-cloudbees - wdyt?

File: content/en/docs/resources/guides/managing-jx/old/create-custom-builder.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ aliases:
   - /getting-started/create-custom-builder
 ---
 
-In Jenkins X, it is possible to create your custom Builders (aka [POD templates](https://github.com/jenkinsci/kubernetes-plugin)) or overwrite existing onces. You just need to base your Docker
-image on this [builder-base](https://github.com/jenkins-x/jenkins-x-builders-base/blob/master/Dockerfile.common) image.
-These images contain a number of pre-installed tools which get constantly updated and published to [Docker Hub](https://hub.docker.com/r/jenkinsxio/builder-base/).
+In Jenkins X, using the deprecated static master installation type, it is possible to create your custom Builders (aka [POD templates](https://github.com/jenkinsci/kubernetes-plugin)) or overwrite existing onces.
```
Comment (jha-cloudbees) [inline content/en/docs/resources/guides/managing-jx/old/create-custom-builder.md:14]:
```suggestion
In Jenkins X, using the deprecated static master installation type, it is possible to create your custom Builders (aka [POD templates](https://github.com/jenkinsci/kubernetes-plugin)) or overwrite existing ones.
```

File: content/en/docs/resources/guides/managing-jx/old/create-custom-builder.md [hunk 2]
```diff
@@ -128,11 +127,9 @@ pipeline {
 
 ## Overwrite existing Builders
 
-Jenkins X comes with a number of [pre-installed builders](https://raw.githubusercontent.com/jenkins-x/jenkins-x-platform/master/jenkins-x-platform/values.yaml)
-which you can overwrite if required during installation or upgrade.
+Jenkins X comes with a number of [pre-installed builders](https://raw.githubusercontent.com/jenkins-x/jenkins-x-platform/master/jenkins-x-platform/values.yaml) which you can overwrite if required during installation or upgrade.
 
-You just need to build your custom image either based on [builder-base](https://github.com/jenkins-x/jenkins-x-builders-base/blob/master/Dockerfile.common)
-image or the [builder image](https://hub.docker.com/u/jenkinsxio/) you want to overwrite. See more details above.
+You just need to build your custom image either based on [builder-base](https://github.com/jenkins-x/jenkins-x-builders-base/blob/master/Dockerfile.common) image or the [builder image](https://hub.docker.com/u/jenkinsxio/) you want to overwrite. See more details above.
```
Comment (jha-cloudbees) [inline content/en/docs/resources/guides/managing-jx/old/create-custom-builder.md:132]:
```suggestion
You just need to build your custom image either based on the [builder-base](https://github.com/jenkins-x/jenkins-x-builders-base/blob/master/Dockerfile.common) image or the [builder image](https://hub.docker.com/u/jenkinsxio/) you want to overwrite. See more details above.
```

File: content/en/docs/resources/guides/managing-jx/old/create-custom-builder.md (hunk 1 above)
Comment (hferentschik) [reply content/en/docs/resources/guides/managing-jx/old/create-custom-builder.md:14]: thanks :-)

**Comment by hferentschik** [comment]
> lgtm. just a couple fixes.

thanks, fixed.

**Comment by hferentschik** [comment]
@ethangj, could you add your lgtm?

**Pull request merged**